### ISSUE TITLE
Help with BIOMOD_xxx - [short question here]

### ISSUE THREAD

**Issue by zali37** [biomodhub/biomod2#690]
_Please make sure to close the issue once you consider it as solved_
_Please use screenshots only when you cannot copy-paste the object, e.g. for figures or maps_

**Context and question**
Hello, Biomod team. This question follows on from some others I asked recently, but I decided to start a new thread. Now that I’ve managed to run the model, I’m having further trouble plotting the response curves for the individual models. The problem is that, given the way I constructed the models, none of them includes `allData_allRun`, so I don’t know how to select a single model that includes all the pseudo-absence sets and all the runs. I’d also like to know how I can include the standard deviation in the response curves for the different algorithms.

**Code used**
bibronii_format <- BIOMOD_FormatingData(
  resp.name = "bibronii",
  resp.var = bibronii[,"presence"],
  resp.xy = bibronii[, c("lon", "lat")],
  expl.var = envs,
  data.type = "binary",
  PA.nb.rep = 5,
  PA.nb.absences = 200,
  PA.strategy = "disk",
  PA.dist.min = -1,
  PA.dist.max = -3,
  filter.raster = T,
  na.rm = T,
  seed.val = 4
)
-=-=-=-=-=-=-=-=-=-=-=-=-=-=-= BIOMOD.formated.data -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=

Modeling directory (dir.name) : C:/DATOS/ENM/diplolaemus

Modeled species (sp.name) : bibronii

	 131 presences,  0 true absences and  981 undefined points in dataset

	 7 explanatory variables

   clip_bio3       clip_bio6        clip_bio8        clip_bio9     
 Min.   :36.50   Min.   :-9.632   Min.   :-4.080   Min.   :-5.216  
 1st Qu.:46.15   1st Qu.:-3.418   1st Qu.: 2.997   1st Qu.: 6.950  
 Median :46.63   Median :-2.040   Median : 4.896   Median :10.755  
 Mean   :46.42   Mean   :-2.076   Mean   : 4.977   Mean   :10.163  
 3rd Qu.:47.20   3rd Qu.:-0.296   3rd Qu.: 6.667   3rd Qu.:13.634  
 Max.   :53.43   Max.   : 2.940   Max.   :14.150   Max.   :18.197  
   clip_bio14       clip_bio15     pedregal_continuoFINAL
 Min.   :  3.00   Min.   : 8.315   Min.   :0.0000000     
 1st Qu.:  9.00   1st Qu.:21.079   1st Qu.:0.0000000     
 Median : 11.00   Median :26.306   Median :0.0002987     
 Mean   : 26.21   Mean   :27.205   Mean   :0.4162051     
 3rd Qu.: 21.00   3rd Qu.:32.323   3rd Qu.:0.3285952     
 Max.   :179.00   Max.   :59.618   Max.   :4.6952438     

 5 Pseudo Absences dataset available ( PA1, PA2, PA3, PA4, PA5 ) with  
200 (PA1, PA2, PA3, PA4, PA5) pseudo absences

-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
bibronii_opt <- bm_ModelingOptions(
  data.type = "binary",
  models=c("SRE","RF", "GAM.gam.gam","ANN","GLM","MAXENT"),
  bm.format = bibronii_format,
  strategy = "user.defined",
  user.base = "bigboss",
  calib.lines = cv.k,
  user.val = list(SRE.binary.biomod2.bm_SRE = opt.sre,
                  RF.binary.randomForest.randomForest = user.rf,
                  GAM.binary.gam.gam = tuned.gam,
                  ANN.binary.nnet.nnet = tuned.ann,
                  GLM.binary.stats.glm = user.GLM,
                  MAXENT.binary.MAXENT.MAXENT = tuned.maxent)
)

Of all these models, I’ve narrowed it down to ANN, RF and GLM.

bibronii_modelo <- BIOMOD_Modeling(
  bm.format = bibronii_format,
  modeling.id = "solo_bibronii",
  models = c("ANN","RF","GLM"),
  CV.strategy = "user.defined",
  CV.user.table = cv.k,
  CV.perc = 0.8,
  var.import = 3,
  OPT.strategy = "user.defined",
  OPT.user = bibronii_opt,
  metric.eval = c("TSS"),
  seed.val = 4
)

<html>
<body>

-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-= BIOMOD.models.out -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=  Modeling directory (dir.name) : C:/DATOS/ENM/diplolaemus  Modeled species (sp.name) : bibronii  Modeling id (modeling.id) : solo_bibronii  Explanatory variables (expl.var.names) : clip_bio3 clip_bio6 clip_bio8 clip_bio9  clip_bio14 clip_bio15 pedregal_continuoFINAL   Computed models (models.computed) :  bibronii_PA1_RUN1_ANN bibronii_PA1_RUN1_RF  bibronii_PA1_RUN1_GLM bibronii_PA1_RUN2_ANN bibronii_PA1_RUN2_RF  bibronii_PA1_RUN2_GLM bibronii_PA1_RUN3_ANN bibronii_PA1_RUN3_RF  bibronii_PA1_RUN3_GLM bibronii_PA1_RUN4_ANN bibronii_PA1_RUN4_RF  bibronii_PA1_RUN4_GLM bibronii_PA2_RUN1_ANN bibronii_PA2_RUN1_RF  bibronii_PA2_RUN1_GLM bibronii_PA2_RUN2_ANN bibronii_PA2_RUN2_RF  bibronii_PA2_RUN2_GLM bibronii_PA2_RUN3_ANN bibronii_PA2_RUN3_RF  bibronii_PA2_RUN3_GLM bibronii_PA2_RUN4_ANN bibronii_PA2_RUN4_RF  bibronii_PA2_RUN4_GLM bibronii_PA3_RUN1_ANN bibronii_PA3_RUN1_RF  bibronii_PA3_RUN1_GLM bibronii_PA3_RUN2_ANN bibronii_PA3_RUN2_RF  bibronii_PA3_RUN2_GLM bibronii_PA3_RUN3_ANN bibronii_PA3_RUN3_RF  bibronii_PA3_RUN3_GLM bibronii_PA3_RUN4_ANN bibronii_PA3_RUN4_RF  bibronii_PA3_RUN4_GLM bibronii_PA4_RUN1_ANN bibronii_PA4_RUN1_RF  bibronii_PA4_RUN1_GLM bibronii_PA4_RUN2_ANN bibronii_PA4_RUN2_RF  bibronii_PA4_RUN2_GLM bibronii_PA4_RUN3_ANN bibronii_PA4_RUN3_RF  bibronii_PA4_RUN3_GLM bibronii_PA4_RUN4_ANN bibronii_PA4_RUN4_RF  bibronii_PA4_RUN4_GLM bibronii_PA5_RUN1_ANN bibronii_PA5_RUN1_RF  bibronii_PA5_RUN1_GLM bibronii_PA5_RUN2_ANN bibronii_PA5_RUN2_RF  bibronii_PA5_RUN2_GLM bibronii_PA5_RUN3_ANN bibronii_PA5_RUN3_RF  bibronii_PA5_RUN3_GLM bibronii_PA5_RUN4_ANN bibronii_PA5_RUN4_RF  bibronii_PA5_RUN4_GLM  Failed models (models.failed) :  none  -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
--
 
> | >
>

</body>
</html>

```
``` 

**Other outputs** _(optional)_
If you have questions about a figure or maps, you can add a screenshot here.

**Environment Information**
Please paste the output of `sessionInfo()` in your current R session below.
```
# paste output of sessionInfo() here
```
Thank you for everything
Natalia

**Comment by zali37** [comment]
Thank you so much Maya.
